src/js/multiple-choice.js

```diff
@@ -21,3 +21,33 @@ const handleOptionButtonClick = (event) => {
 optionButtons.forEach((button) => {
   button.addEventListener('click', handleOptionButtonClick);
 });
+
+const randomizeAnswers = () => {
+  // grab and group multiple choice items by needs-js and no-js  pairs
+  const multipleChoiceItems = document.querySelectorAll(".cmp-multiple-choice__item");
+  const needsJSNoJSPairs = [];
+  for (const [index, element] of multipleChoiceItems.entries()) {
+    if (index % 2 === 0) {
+      needsJSNoJSPairs.push({
+        needsJS: element,
+        noJS: multipleChoiceItems[index + 1],
+      });
+    }
+  }
```

Comment:
Since we know that JS is available if we're getting to this point, we don't actually need to preserve the `noJS` elements, which will let us target the elements we want more directly.

```js
const multipleChoiceItems = document.querySelectorAll('.cmp-multiple-choice__item[data-needs-js]');
```

No `for` loop necessary now!